### PR TITLE
pattern matching simplification

### PR DESCRIPTION
**Pull request by ghost**
this pr fixes issue #5

### CODE REVIEW

File: src/utils/scripts.rs
Comment (ShashwatAgrawal20) [inline src/utils/scripts.rs:1]: I don't quite understand why did you commented the whole file?

File: migration/src/m20220101_000001_create_table.rs [hunk 1]
```diff
@@ -57,7 +57,7 @@ impl MigrationTrait for Migration {
 enum User {
     Table,
     Id,
-    UserName,
+    Name,
```
Comment (ShashwatAgrawal20) [inline migration/src/m20220101_000001_create_table.rs]: And why did you changed this? is that some clippy thing?

**Comment by Sidharth-Singh10** [comment]
Hey @oebelus ,Currently clippy warnings are ignored. You can continue to work only within the scope of the issue.

File: src/utils/scripts.rs
Comment (Sidharth-Singh10) [reply src/utils/scripts.rs:1]: Clippy warnings. Currently we ignore Clippy warnings but @oebelus tried to fix them.

Comment (ShashwatAgrawal20) [reply src/utils/scripts.rs:1]: ohh fair enough.

File: migration/src/m20220101_000001_create_table.rs (hunk 1 above)
Comment (ghost) [reply migration/src/m20220101_000001_create_table.rs]: It's also a warning from Clipper about naming conventions for consistency with Rust's guidelines

**Comment by ghost** [comment]
In the issue, it was asked to run `cargo clippy` and it threw warnings regarding other pattern matching and naming conventions,

**Comment by ghost** [comment]
Should I re-edit the code to only simplify Err(_) ?

**Comment by Wahid7852** [comment]
> Should I re-edit the code to only simplify Err(_) ?

yes please, that'd be optimal
we shall try to fix clippy warning later

**Comment by Wahid7852** [comment]
You can just reset --hard to the commit the repo was before you commited to it and then do your changes on a clean new base @oebelus

**Pull request closed without merging**